internal/cli/create/create.go

```diff
@@ -111,14 +110,9 @@ func (c *Command) loadModule() *Module {
 	if module.Name == "" {
 		// Try inferring the module name from the directory
 		module.Name = mod.Infer(c.absDir)
+		// if can't infer then default it to `changeme`
 		if module.Name == "" {
-			// Fail that you need to pass in a module path
-			c.bail.Bail(format.Errorf(`
-			Unable to infer a module name. Try again using the module <path> name.
-
-			For example,
-				bud create --module=github.com/my/app %s
-		`, c.Dir))
+			module.Name = "changeme"
```

Comment:
I think we should make it `change.me` because `changeme` isn't a valid module name. This doesn't appear to have any ramifications right now, but I don't see a big deal with generating a valid name. See below for an example:

```go
package main

import (
	"fmt"
	"os"

	"golang.org/x/mod/module"
)

func main() {
	if len(os.Args) <= 1 {
		fmt.Println("no args")
		return
	}
	modulePath := os.Args[1]
	err := module.CheckPath(modulePath)
	if err != nil {
		fmt.Printf("invalid path %s: %s\n", modulePath, err)
		os.Exit(1)
	}
	fmt.Printf("valid path %s\n", modulePath)
}
```